app/lib/rocketchat.js

```diff
@@ -614,11 +614,17 @@ const RocketChat = {
 		// RC 0.64.0
 		return this.sdk.post('rooms.favorite', { roomId, favorite });
 	},
-	toggleRead(read, roomId) {
+	toggleRead(read, roomId, firstUnreadMessageId) {
```

Comment:
Can you switch this signature to `toggleRead({ read = true, rid, messageId })`?
This way is much more readable.

#### Before:
```jsx
RocketChat.toggleRead(true, undefined, actionMessage._id);
```

#### After:
```jsx
RocketChat.toggleRead({ messageId : actionMessage._id });
```

Reply:
And on https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/views/RoomsListView/index.js#L392:

#### Before:
```jsx
const result = await RocketChat.toggleRead(isRead, rid);
```

#### After:
```jsx
const result = await RocketChat.toggleRead({ rid, read: isRead });
```